pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
FYI @BhaswatiRoy, we observe sentence capitalization for headers in our docs. **Please update all headers accordingly** :)